pkg/profiler/cpu.go

```diff
@@ -314,13 +315,45 @@ func (p *CPUProfiler) Labels() model.LabelSet {
 	return labels
 }
 
+var configPaths = []string{
+	"/proc/config.gz",
+	"/boot/config",
+	"/boot/config-%s",
+}
+
 func (p *CPUProfiler) Run(ctx context.Context) error {
 	level.Debug(p.logger).Log("msg", "starting cpu profiler")
 
 	p.mtx.Lock()
 	ctx, p.cancel = context.WithCancel(ctx)
 	p.mtx.Unlock()
 
+	isContainer, err := kconfig.IsInContainer()
```

Comment:
Let's move these checks to "cmd/parca-agent/main". 
And just stop the program at the start-up time if it's a no go for us.

Reply:
Just to be double sure, you mean here right after initializing parca-agent, right? https://github.com/parca-dev/parca-agent/blob/main/cmd/parca-agent/main.go#L124

Reply:
Yes, exactly!